### PR TITLE
ircii 20240111

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ircii
    * The homepage URL http://www.eterna.com.au/ircii/ is not reachable
```

File: Formula/i/ircii.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Ircii < Formula
   desc "IRC and ICB client"
-  homepage "http://www.eterna.com.au/ircii/"
-  url "https://ircii.warped.com/ircii-20210314.tar.bz2"
-  mirror "https://deb.debian.org/debian/pool/main/i/ircii/ircii_20210314.orig.tar.bz2"
-  sha256 "866f2b847daed3d70859f208f7cb0f20b58c0933b2159f7ff92a68c518d393a9"
+  homepage "https://web.archive.org/web/20231024192652/http://eterna.com.au/ircii/"
```
Comment (chenrui333) [inline Formula/i/ircii.rb:3]: reaching out to the upstream about the site issue.

**Review by chenrui333 — APPROVED** [review]

File: Formula/i/ircii.rb (hunk 1 above)
Comment (p-linnane) [reply Formula/i/ircii.rb:3]: We should add a comment explaining that it should be temporary and that we've reached out.

Comment (chenrui333) [reply Formula/i/ircii.rb:4]: just added an note, let me know if that works.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**